app/controllers/api/base_controller.rb

```diff
@@ -0,0 +1,21 @@
+require 'doorkeeper_auth'
+
+class Api::BaseController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  include DoorkeeperAuth
+
+  api_accessible! true
+
+  def paginate(scope)
+    if scope.is_a?(Array)
+      Kaminari.paginate_array(scope).page(page).per(1000)
```

Comment:
having an array here seems like a sign of trouble/something gone wrong ... maybe blow up ?